packages/react-server/src/ReactFlightServer.js

```diff
@@ -1677,8 +1677,11 @@ function emitErrorChunk(
         message = String(error.message);
         // eslint-disable-next-line react-internal/safe-string-coercion
         stack = String(error.stack);
+      } else if (typeof error === 'object' && error !== null) {
+        message = 'Error: ' + describeObjectForErrorMessage(error);
       } else {
-        message = 'Error: ' + (error: any);
```

Comment:
This prefix actually doesn't really make sense because this prefix already comes as part of the `.stack` property with the constructor's name.